packages/webapp/src/stories/Table/TableCells.stories.jsx

```diff
@@ -68,45 +70,156 @@ const getFakeColumns = () => {
   ];
 };
 
+const getFakeColumnsTwo = () => {
+  return [
+    {
+      id: 'date',
+      label: 'Date',
+      format: (d) => (
+        <Cell
+          kind={CellKind.ICON_TEXT}
+          iconName={'CALENDAR'}
+          text={getIntlDate(d.date)}
+          className={styles.dateCell}
+        />
+      ),
+    },
+    {
+      id: 'taskStatus',
+      label: 'Task Status',
+      format: (d) => (
+        <Cell
+          kind={CellKind.TASK_STATUS_INDICATOR_PILL}
+          color={d.taskStatus}
+          label={d.taskStatus.toUpperCase()}
```

Comment:
Really small detail but the label is in lowecase in the Figma so I'd keep it that way! Although if we have the component get the translation for the status as I mentioned in my other comment this prop won't be needed anyway so moot point

Reply:
Good point, I was reusing existing implementations of TaskLabel I can probably find a function that will capitalize only the first letter (https://stackoverflow.com/questions/1026069/how-do-i-make-the-first-letter-of-a-string-uppercase-in-javascript). But otherwise is it worth opting for a whole new translation?